### ISSUE TITLE
google_maps_flutter web V 0.5.27

### ISSUE THREAD

**Issue by isaacolajos** [flutter/flutter#56122]
Web compatibility is lit up (only as of this version) but it doesn't seem to work and there is no documentation referencing web setup. 

Is this a misprint or is there support for flutter web in this version? Running in beta channel. 

```console
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel beta, v1.17.0-3.2.pre, on Microsoft Windows [Version 10.0.18363.778], locale en-CA)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Chrome - develop for the web
[√] Android Studio (version 3.6)
[√] VS Code (version 1.43.0)
[√] Connected device (3 available)

• No issues found!


```

**Comment by iapicca** [comment]
Hi @isaacolajos 
can you please provide your `flutter run --verbose` 
and your `pubspec.yaml` 
or if possible a reproducible minimal code sample.
Thank you

**Comment by isaacolajos** [comment]
Its running android no issue but when trying to run the maps screen on the web its white. 





name: fact40
description: Factory 4.0

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
version: 1.0.0+1

environment:
  sdk: ">=2.0.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.2

  path_provider: ^1.6.5
  http: ^0.12.0
  uuid: any
  image_picker: ^0.6.0+15
  xml: ^3.4.1
  crypto: ^2.0.6
  pointycastle: ^1.0.0-rc3
  connectivity: ^0.4.5+1
  charts_flutter: ^0.8.0
  datetime_picker_formfield: ^0.1.7
  image: ^2.0.5
  photo_view: ^0.6.0
  flutter_nfc_reader: ^0.1.0
  device_id: ^0.2.0
  location: ^2.3.5
  firebase_ml_vision: 0.9.3+3
  expressions: ^0.1.5
  flutter_launcher_icons: any
  hive: any
  syncfusion_flutter_charts: ^18.1.45
  rflutter_alert: ^1.0.3
  file_picker: ^1.6.2
  google_maps_flutter: ^0.5.27
  flutter_webview_plugin: ^0.3.11

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter_icons:
  image_path: "images/icon.png"
  android: true
  ios: true

fonts:
  - family: Roboto
    fonts:
      - asset: fonts/Roboto/Roboto-Regular.ttf
        

# For information on the generic Dart part of this file, see the
# following page: https://dart.dev/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:

  # The following line ensures that the Material Icons font is
  # included with your application, so that you can use the icons in
  # the material Icons class.
  uses-material-design: true

  # To add assets to your application, add an assets section, like this:
  # assets:
  assets:
    - images/a1px.png
    - images/splash.png
    - images/logo.png
    - images/icon.png
    - images/logo-light.png
    - images/poly_stock.jpg


  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.dev/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.dev/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  # fonts:
  #   - family: Schyler
  #     fonts:
  #       - asset: fonts/Schyler-Regular.ttf
  #       - asset: fonts/Schyler-Italic.ttf
  #         style: italic
  #   - family: Trajan Pro
  #     fonts:
  #       - asset: fonts/TrajanPro.ttf
  #       - asset: fonts/TrajanPro_Bold.ttf
  #         weight: 700
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.dev/custom-fonts/#from-packages


C:\Users\isaac\Documents\Developer\GitHub\core-client>flutter run --verbose
[  +63 ms] executing: [C:\Users\isaac\Documents\Developer\flutter\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +145 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +4 ms] e6b34c2b5c96bb95325269a29a84e83ed8909b5f
[   +2 ms] executing: [C:\Users\isaac\Documents\Developer\flutter\flutter/] git tag --contains HEAD
[ +629 ms] Exit code 0 from: git tag --contains HEAD
[   +1 ms] 1.17.0-3.4.pre
[  +28 ms] executing: [C:\Users\isaac\Documents\Developer\flutter\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +90 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/beta
[   +1 ms] executing: [C:\Users\isaac\Documents\Developer\flutter\flutter/] git ls-remote --get-url origin
[  +82 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms] https://github.com/flutter/flutter.git
[ +257 ms] executing: [C:\Users\isaac\Documents\Developer\flutter\flutter/] git rev-parse --abbrev-ref HEAD
[ +101 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +2 ms] beta
[ +116 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +12 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +55 ms] executing: C:\Users\isaac\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[ +103 ms] List of devices attached
[  +18 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[  +11 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[   +2 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +171 ms] More than one device connected; please specify a device with the '-d <deviceId>' flag, or use '-d all' to act on all devices.
[  +56 ms] Chrome     • chrome     • web-javascript • Google Chrome 81.0.4044.122
[   +4 ms] Web Server • web-server • web-javascript • Flutter Tools
[  +27 ms] "flutter run" took 715ms.

#0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
#1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:336:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:702:11)
<asynchronous suspension>
#3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:619:33)
<asynchronous suspension>
#4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart)
#5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:29)
#6      _rootRun (dart:async/zone.dart:1184:13)
#7      _CustomZone.run (dart:async/zone.dart:1077:19)
#8      _runZoned (dart:async/zone.dart:1619:10)
#9      runZoned (dart:async/zone.dart:1539:10)
#10     AppContext.run (package:flutter_tools/src/base/context.dart:149:18)
#11     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:609:20)
#12     CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#13     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:339:21)
<asynchronous suspension>
#14     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart)
#15     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:29)
#16     _rootRun (dart:async/zone.dart:1184:13)
#17     _CustomZone.run (dart:async/zone.dart:1077:19)
#18     _runZoned (dart:async/zone.dart:1619:10)
#19     runZoned (dart:async/zone.dart:1539:10)
#20     AppContext.run (package:flutter_tools/src/base/context.dart:149:18)
#21     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:288:19)
#22     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#23     new Future.sync (dart:async/future.dart:224:31)
#24     CommandRunner.run (package:args/command_runner.dart:112:14)
#25     FlutterCommandRunner.run (package:flutter_tools/src/runner/flutter_command_runner.dart:231:18)
#26     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:63:22)
#27     _rootRun (dart:async/zone.dart:1184:13)
#28     _CustomZone.run (dart:async/zone.dart:1077:19)
#29     _runZoned (dart:async/zone.dart:1619:10)
#30     runZonedGuarded (dart:async/zone.dart:1608:12)
#31     runZoned (dart:async/zone.dart:1536:12)
#32     run.<anonymous closure> (package:flutter_tools/runner.dart:61:18)
<asynchronous suspension>
#33     run.<anonymous closure> (package:flutter_tools/runner.dart)
#34     runInContext.runnerWrapper (package:flutter_tools/src/context_runner.dart:64:18)
<asynchronous suspension>
#35     runInContext.runnerWrapper (package:flutter_tools/src/context_runner.dart)
#36     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:29)
#37     _rootRun (dart:async/zone.dart:1184:13)
#38     _CustomZone.run (dart:async/zone.dart:1077:19)
#39     _runZoned (dart:async/zone.dart:1619:10)
#40     runZoned (dart:async/zone.dart:1539:10)
#41     AppContext.run (package:flutter_tools/src/base/context.dart:149:18)
#42     runInContext (package:flutter_tools/src/context_runner.dart:67:24)
#43     run (package:flutter_tools/runner.dart:48:10)
#44     main (package:flutter_tools/executable.dart:69:9)
#45     main (file:///C:/Users/isaac/Documents/Developer/flutter/flutter/packages/flutter_tools/bin/flutter_tools.dart:8:3)
#46     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#47     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)




C:\Users\isaac\Documents\Developer\GitHub\core-client>

**Comment by TahaTesser** [comment]
Hi @isaacolajos 
Looks like `google_maps_flutter ` isn't supported for web yet
Closing for [#45292](https://github.com/flutter/flutter/issues/45292)

Closing, as this isn't an issue with `google_maps_flutter ` itself,
if you disagree please write in the comments and I will reopen it 
Thank you

**Comment by isaacolajos** [comment]
The only reason I ask is because web is lit up of the official package page. Just may want to change that to avoid further confusion, thanks!

**Comment by ditman** [comment]
This [code](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter/pubspec.yaml#L31-L32) needs to be reverted. I pushed it by mistake, and it wasn't noticed in PR.

Thanks for reporting @isaacolajos !

**Comment by ditman** [comment]
Published version [`0.5.27+1`](https://pub.dev/packages/google_maps_flutter) that doesn't publicize web support.
